src/js/components/Form/stories/typescript/CustomTheme.tsx

```diff
@@ -38,7 +39,5 @@ const CustomFormField = () => (
 );
 
 if (!isChromatic()) {
-  storiesOf('TypeScript/Form', module).add('Custom Theme', () => (
-    <CustomFormField />
-  ));
+  storiesOf('Form', module).add('Custom Theme', () => <CustomFormField />);
```

Comment:
If there is no js duplication we should enable chromatic.

Reply:
I realied the `CustomTheme.tsx` story wasn't testing Form but instead just testing FormField so I moved this test to be under FormField instead of Form. I also enabled chromatic testing on it